pages/docs/reference/js-modules.md

```diff
@@ -19,68 +19,77 @@ the list of available options:
 4. Unified Module Definitions (UMD), which is compatible with both *AMD* and *CommonJS*, and works as "plain"
    when neither *AMD* nor *CommonJS* is available at runtime.
 
+## Targeting the browser
+ 
+If you're targeting the browser, you can specify the desired module type in the `webpackTask` configuration block:
+ 
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-lang="groovy">
+
+```groovy
+kotlin {
+    target {
+        browser {
+            webpackTask {
+                output.libraryTarget = COMMONJS 
+                //commonjs // a shorter alternative
```

Comment:
I don't understand this. We can say that you can set to `libraryTarget` plain string for example
`output.libraryTarget = "commonjs"`